src/int.cr

```diff
@@ -456,6 +456,19 @@ struct Int
     yield ptr, count
   end
 
+  def digits(base = 10)
+    num = abs
+    arr = Array(typeof(self)).new((Math.log(num)/Math.log(base)).ceil.to_i)
```

Comment:
Put spaces in between of `/` (readability-wise), please.

Reply:
Done 👍🏼

Reply:
Cool, thanks!